scrapinghub/legacy.py

```diff
@@ -24,8 +26,6 @@
 
 
 __all__ = ["APIError", "Connection"]
```

Comment:
you need to remove this line as well for `from .legacy import *` to work as we expect